frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -137,6 +126,26 @@ const ManagePolicyPage = ({
     }
   );
 
+  const canAddOrRemovePolicy =
+    isGlobalAdmin || isGlobalMaintainer || isTeamMaintainer || isTeamAdmin;
+
+  const {
+    data: failingPoliciesWebhook,
+    isLoading: isLoadingFailingPoliciesWebhook,
+    refetch: refetchFailingPoliciesWebhook,
+  } = useQuery<IConfigNested, Error, IWebhookFailingPolicies>(
+    ["config"],
+    () => configAPI.loadAll(),
+    {
+      enabled: canAddOrRemovePolicy,
+      select: (data: IConfigNested) =>
+        data.webhook_settings.failing_policies_webhook,
+      onSuccess: (data) => {
```

Comment:
Ooo good point! Maybe we do `setConfig` whenever we `configAPI.update()`?

Reply:
That makes sense too. Does the `update` method return the full `config` in the response?

Reply:
Cool, I think it wouldn't hurt to update it there too.

Reply:
Great idea! Agreed that a pattern of always setting context when we retrieve new data makes sense. Otherwise we run the risk of data getting out of sync.

Reply:
@lukeheath I'll add that into the policy automation update here and the software vulnerability automation in my other PR.

Reply:
Hmm, it looks like our context config still expects the flattened config file. Does it make sense for us to add a new `configNested, setConfigNested` context that we can use in parallel until we decide to remove all the flat config usage in the app? If so, I can do that in my PR for the software tab?